CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ add_subdirectory(examples)
 ################################################################################
 install(
   DIRECTORY hpxio
-  DESTINATION include/hpxio
+  DESTINATION include
```

Comment:
Are you installing all of your header files directly into `.../include/`? Wouldn't it be better to leave the hpxio header files in a separate sub-directory which would allow to include then as:

    #include <hpxio/foo.hpp>

?